src/ir.cc

```diff
@@ -703,7 +703,8 @@ void Var::Destroy() {
   }
 }
 
-uint8_t ElemSegment::GetFlags(const Module* module) const {
+uint8_t ElemSegment::GetFlags(const Module* module,
+                              bool function_references_enabled) const {
```

Comment:
It seems a little odd that that we don't have any other features in `ir.cc`, I guess we have gotten away to a feature-agnostic IR up until this point.

Reply:
Normally, new features are backward compatible, so you don't need flags for them (the parser will check the feature). This is probably the first exception.